torch/onnx/__init__.py

```diff
@@ -67,8 +67,6 @@
     "TrainingMode",
     "TensorProtoDataType",
     "JitScalarType",
-    # Classes
-    "SymbolicContext",
```

Comment:
Removed deprecated class from `__all__`. it is still accessible and existing code will not break.